src/containers/pages/InterventionPlan/NewPlan/General/index.tsx

```diff
@@ -43,8 +89,11 @@ const NewPlan = () => {
       <h3 className="mb-3 page-title">{pageTitle}</h3>
       <Row>
         <Col md={8} id="planform-col-container">
-          <PlanForm {...{ formHandler: formValuesHandler }} />
+          <PlanForm {...planFormProps} key={formValues.interventionType} />
         </Col>
+        {/* ISSUE - TODO - for FI plans we cannot select the country jurisdiction when creating the plan since this component
+        below depends on having a focus area  whose value is set by using a cascade select in the jurisdiction select field
+        of planForm. it would have to be moved or changed.*/}
```

Comment:
I don't understand why for FI plans we cannot simply have the user select a country (root jurisdiction) as usual.  What is preventing that?

Reply:
I have handled this in  #963